birdset/datamodule/components/transforms.py

```diff
@@ -445,7 +445,7 @@ def _waveform_scaling(self, audio_augmented, attention_mask):
         return audio_augmented
    
     def _prepare_call(self):
-        if self.mode in ("valid", "test", "predict"):
+        if self.mode in ("test", "predict"):#, "valid"): ##!! important
```

Comment:
valid, test, predict should be the default setting.

Reply:
```suggestion
        if self.mode in ("test", "predict", "valid"):
```

Reply:
```suggestion
        if self.mode in ("test", "predict", "valid"):
```